web/scripts/embed.dart

```diff
@@ -4,8 +4,8 @@
 
 // Currently not in use
 
-import 'package:dart_pad/embed/embed.dart' as embed;
-import 'package:dart_pad/polymer/polymer.dart';
+import '../../lib/embed/embed.dart' as embed;
+import '../../lib/polymer/polymer.dart';
```

Comment:
We need to update pub to find the correct packages within the scripts directory. 
The change was auto-generated when I moved the files around into the script subdirectory.